packaging/standalone/standalone-enterprise/src/main/distribution/text/enterprise/conf/neo4j.conf

```diff
@@ -294,6 +296,159 @@ dbms.directories.import=import
+# and does not need to have access rights to Neo4j or any other systems.
+#dbms.security.ldap.authorization.use_system_account=false
+
+# An LDAP system account username to use for authorization searches when
+# `dbms.security.ldap.authorization.use_system_account` is `true`.
+# Note that the `dbms.security.ldap.user_dn_template` will not be applied to this username,
+# so you may have to specify a full DN.
+#dbms.security.ldap.system_username=
+
+# An LDAP system account password to use for authorization searches when
+# `dbms.security.ldap.authorization.use_system_account` is `true`.
+#dbms.security.ldap.system_password=
+
+# The name of the base object or named context to search for user objects when LDAP authorization is enabled.
+# A common case is that this matches the last part of `dbms.security.ldap.user_dn_template`,
+# e.g. `ou=users,dc=example,dc=com`.
+#dbms.security.ldap.authorization.user_search_base=
+
+# The LDAP search filter to search for a user principal when LDAP authorization is
+# enabled. The filter should contain the placeholder token {0} which will be substituted for the
+# user principal.
+#dbms.security.ldap.authorization.user_search_filter=(&(objectClass=*)(uid={0}))
+
+# A list of attribute names on a user object that contains groups to be used for mapping to roles
+# when LDAP authorization is enabled.
+#dbms.security.ldap.authorization.group_membership_attributes=memberOf
+
+# An authorization mapping from LDAP group names to Neo4j role names.
+# The map should be formatted as a semicolon separated list of key-value pairs, where the
+# key is the LDAP group name and the value is a comma separated list of corresponding role names.
+# E.g. group1=role1;group2=role2;group3=role3,role4,role5
+#dbms.security.ldap.authorization.group_to_role_mapping=
+
+# You could also use whitespaces and quotes around group names to make this mapping more readable,
+# e.g: dbms.security.ldap.authorization.group_to_role_mapping=\
+#          "cn=Neo4j Read Only,cn=users,dc=example,dc=com"      = reader;    \
+#          "cn=Neo4j Read-Write,cn=users,dc=example,dc=com"     = publisher; \
+#          "cn=Neo4j Schema Manager,cn=users,dc=example,dc=com" = architect; \
+#          "cn=Neo4j Administrator,cn=users,dc=example,dc=com"  = admin
+
+#================================================
+# Auth Cache Configuration
+#================================================
+
+# The time to live (TTL) for cached authentication and authorization info when using
+# external auth providers (ldap or plugin). Setting the TTL to 0 will
+# disable auth caching.
+#dbms.security.auth_cache_ttl=10m
+
+# The maximum capacity for authentication and authorization caches (respectively).
+#dbms.security.auth_cache_max_capacity=10000
+
+#================================================
+# Security Log Configuration
+#================================================
+# The security log is always enabled and resides in `logs/security.log`.
```

Comment:
I think I would put the logging settings in to the main logging section. `log_successful_authentication` could go in the top bit of the security section without a heading.